src/evo/simplifiedmns.cpp

```diff
@@ -88,13 +87,9 @@ uint256 CSimplifiedMNList::CalcMerkleRoot(bool* pmutated) const
     return ComputeMerkleRoot(leaves, pmutated);
 }
 
-CSimplifiedMNListDiff::CSimplifiedMNListDiff()
-{
-}
+CSimplifiedMNListDiff::CSimplifiedMNListDiff() = default;
```

Comment:
These 2 should be dropped here and defined in `.h` like others I guess

Reply:
Why should they be in the header? (Whenever I try to include it in the header I get link errors) (see two test commits here https://github.com/PastaPastaPasta/dash/commits/tidy-evo-2)

Reply:
Hmm... right, can't easily do this for `CSimplifiedMNListDiff` because of `CFinalCommitment` which is not fully defined at this moment which requires including a header which in its turn creates circular dependencies all of which can be fixed like this b85c9b91de but... moving it was probably not a good idea in the first place, so nvm :)